rascal-vscode-extension/src/test/vscode-suite/ide.test.ts

```diff
@@ -188,6 +188,31 @@ describe('IDE', function () {
         expect(editorText).to.contain("i -2");
     });
 
+    it("renaming files works", async() => {
```

Comment:
nice test :+1:

Reply:
Had to add this, since I think consistently verifying at least basic functionality of this callback is difficult otherwise.